RELEASE.md

```diff
@@ -232,6 +232,31 @@ Please also make sure to add milestone target to the PR/issue, especially if it
 
 **NOTE**: The cherry pick process is not an invitation to add new features, it is mainly there to fix regressions
 
+#### How to do Cherry Picking
+
+You can now use `pytorchbot` to cherry pick a PR that has been committed
+to the main branch using `@pytorchbot cherry-pick` command as follows.
```

Comment:
Please list all the repos this feature is enabled in.

Reply:
Good point.  As the bot is only available in PyTorch atm, this implies that we only have this one PyTorch.  Let me make that point clearer.